sql/moz-fx-data-shared-prod/static/key_tentpole_dates/data.csv

```diff
@@ -0,0 +1,113 @@
+Key Tentpole	Start Date	End Date
+New Year's  	January 1	January 1
```

Comment:
Can you put all the dates in `YYYY-MM-DD` format?

Reply:
@curtismorales Can do!